docs/disable_resources.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Disabling Resources Through Tilt
+description: "Use Tilt's new feature to disable and enable resources through the UI. Manage what resources you have up and running more seamlessly."
+layout: docs
+sidebar: guides
+---
+
+> <span class="pill-tag">Beta</span> This feature is in beta. It's turned off by default, and breaking changes could be introduced while under development.
+
+Through Tilt’s UI Dashboard, you can enable and disable resources that are available in your Tilt session.
+
+Disabling a resource will stop the running process (if any) and delete any objects owned by that resource. It’s the equivalent of running `tilt down` on a specific resource. Enabling a resource will create, build, deploy, and start any processes for that resource.
+
+Depending on your `Tiltfile` setup, it may be important to note:
+* Disabling and enabling a resource doesn’t take into account any of its resource dependencies. For example, if resource B depends on resource A, disabling resource A will not disable resource B.
+* If you’re using [`tilt args`](tiltfile_config.html) to define a subset of resources Tilt is running, only the resources that are defined through args will show up in the UI Dashboard. In a future release, we’re planning to display all defined resources from the Tiltfile in the UI.
+
+## How to enable the feature
+While in beta, Disabling Resources feature requires opting in. First, make sure you're on [Tilt v0.23.5+](https://github.com/tilt-dev/tilt/releases). Then, add the following line to your Tiltfile:
+
+```
+enable_feature(“disable_resources”)
```

Comment:
```suggestion
enable_feature("disable_resources")
```

My editor was yelling at me when I copied the example in the current docs.

Reply:
ah, great catch! thanks. 🌟